src/redist/project.json

```diff
@@ -20,7 +21,8 @@
     "Microsoft.Build.Targets": "0.1.0-preview-00032-160824",
     "Microsoft.Build": "0.1.0-preview-00032-160824",
     "System.Runtime.Serialization.Xml": "4.1.0",
-    "NuGet.Build.Tasks": "3.6.0-beta.1.msbuild.1"
+    "NuGet.Build.Tasks": "3.6.0-beta.1.msbuild.1",
+	  "Microsoft.TestPlatform": "15.0.0-preview-*"
```

Comment:
This dependency is specified in "tool_testplatform".  Does it need to be duplicated here?

---
In reply to: [76810686](https://github.com/dotnet/cli/pull/4115#discussion_r76810686) [](ancestors = 76810686)